numjuggler/utils/PartialFormatter.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import print_function, absolute_import, division
```

Comment:
The ``PartialFormatter`` seems to provide some features to the underlying ``string.formatter``. COuld you please describe the features here, and why they are important in numjuggler?

Reply:
The MCNP models may contain entries like {31c}. The previous version of the code failed on this, looking for the kwargs dictionary a field with the name '31c'. The partial formatter ignores these entries. Otherwise you have to clean the original MCNP code from these entries, but this is not good idea: you'd better don't change the original files at all.

Reply:
Just curiosity: the entry like ``{31c}`` is a valid MCNP input syntax, or occasionally can be found within the comments?

Reply:
This entry happens in comments